### ISSUE TITLE
Can we expect an update of Kitsune Magisk?

### ISSUE THREAD

**Issue by BadIDMan** [HuskyDG/magisk-files#174]
I don't know where to ask in proper way so I decided to ask here: can we expect an update of Kitsune Magisk?
Latest Canary Kitsune build is quite old.
R6687BB53-kitsune (27001) was released on July 5th.
Since then there was released few releases of official Magisk and today it is already version 28.0
It would be great to have new Canary Kitsune version with the feature "Action" like it was added to 2708 official Magisk 3 weeks ago: "[App] Add action.sh support to allow modules to define an action triggered from UI"
https://github.com/topjohnwu/Magisk/releases

**Comment by VendorAttestation** [comment]
@HuskyDG 🙏

**Comment by dioxias** [comment]
would be really nice to get A15 graphene support at the same time

**Comment by 1q23lyc45** [comment]
v27.2(27200) with action.sh
https://github.com/1q23lyc45/KitsuneMagisk/releases/tag/v27.2-kitsune-2

**Comment by dioxias** [comment]
> v27.2(27002) with action.sh https://github.com/1q23lyc45/KitsuneMagisk/releases/tag/v27.2-kitsune-2

does this work on graphene A15? either way thanks for working on it!

**Comment by 1q23lyc45** [comment]
> > v27.2(27002) with action.sh https://github.com/1q23lyc45/KitsuneMagisk/releases/tag/v27.2-kitsune-2
> 
> does this work on graphene A15? either way thanks for working on it!

Yes, but you need to install the ReZygisk module separately. The built-in Zygisk is not supported at the moment, and may be supported in the future.
I'm a bit busy and may not be updated in time lately, sorry.
